src/app/accounts/create-account.ts

```diff
@@ -64,12 +71,38 @@ const initializeCreatedAccount = async ({
   account.role = role || "user"
   account.contactEnabled = account.role === "user" || account.role === "editor"
 
+  account.statusHistory = [{ status: config.initialStatus, comment: "Initial Status" }]
+  account.level = config.initialLevel
+
   const updatedAccount = await AccountsRepository().update(account)
   if (updatedAccount instanceof Error) return updatedAccount
 
   return updatedAccount
 }
 
+export const createAccountForDeviceAccount = async ({
+  userId,
+}: {
+  userId: UserId
+}): Promise<Account | RepositoryError> => {
+  const user = await UsersRepository().update({ id: userId })
```

Comment:
not sure to understand this... what are you updating?

Reply:
code was updated to include deviceId. also it looks like the update uses the config below

```
{
  new: true,
  upsert: true,
}
```